sdk/Lusid.Sdk/Client/ApiClient.cs

```diff
@@ -3,7 +3,7 @@
  *
- * The version of the OpenAPI document: 0.11.3754
+ * The version of the OpenAPI document: 0.11.3757
```

Comment:
the is a generated file and shouldn't be checked in

Reply:
good spot. ApiResponse.cs changes should be made in the generator template though as it is generated